lib/jekyll/cache.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require "digest"
+require "fileutils"
+require "pstore"
+
+module Jekyll
+  class Cache
+    extend Forwardable
+
+    # Get an existing named cache, or create a new one if none exists
+    #
+    # name - name of the cache
+    #
+    # Returns nothing.
+    # rubocop:disable Style/ClassVars
+    def initialize(name)
+      throw unless defined? @@safe
```

Comment:
What "catches" this throw here?

Reply:
Nothing. The idea is that `@@safe` must be defined before the cache can be used. `@@safe` is set when Jekyll is initialized. I'm going to change the semantics of this :+1: